integration-tests/contracts/contract_loader.go

```diff
@@ -3,8 +3,7 @@ package contracts
 import (
 	"errors"
 
-	"github.com/smartcontractkit/chainlink-testing-framework/contracts/ethereum"
-	int_ethereum "github.com/smartcontractkit/chainlink/integration-tests/contracts/ethereum"
+	ethereum2 "github.com/smartcontractkit/chainlink/integration-tests/contracts/ethereum"
```

Comment:
Let's stay consistent with naming, switch to `eth_contracts`

Reply:
Renamed to `eth_contracts` everywhere